test/codeSigning/simplifiedSigning.test.ts

```diff
@@ -4,17 +4,17 @@
  * Licensed under the BSD 3-Clause license.
  * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
  */
-import { writeFile, unlink } from 'fs/promises';
+import { unlinkSync, writeFileSync } from 'fs';
```

Comment:
I was unable to `yarn test` because it was unable to locate `fs/proimses`. I tried deleting `node_modules` and `yarn.lock` but nothing seemed to work